src/utils/simpleResponse.ts

```diff
@@ -0,0 +1,16 @@
+import * as msRest from "ms-rest-js";
+
+export default class SimpleResponse {
+  constructor(
+    private readonly httpOperationResponse: msRest.HttpOperationResponse
+  ) {}
+
+  // tslint:disable-next-line:no-any
+  public bodyAsJson(): any {
+    return this.httpOperationResponse.bodyAsJson;
+  }
+
+  public status(): number {
```

Comment:
as I've said above, I would perhaps avoid exposing the status code directly and instead provide simple helper methods (`isOK`, `isNotFound`)